tests/RequestTest.php

```diff
@@ -9,3 +10,46 @@
 
     // dd($response);
 });
+
+it('creates a new instance in Mautic when the model does not have a mautic_id', function () {
+    $mock = mock(\Mautic\Api\Contacts::class)->expect(
+        create: fn (array $attributes) => ['contact' => ['id' => '1337']]
```

Comment:
You don't assert the correct data is sent to Mautic i.e. it uses the `toMauticArray` method.